### PR TITLE
Create theme

### PR DESCRIPTION
**Pull request by aninternetof**
This creates a material-ui theme and applies it to the entire application.

### CODE REVIEW

File: src/components/ProgramList.js [hunk 1]
```diff
@@ -191,8 +191,14 @@ class ProgramList extends Component {
 
     return (
       <>
-        <Button variant="contained" color="primary" href="/mission-control" style={{ marginLeft: '10%' }}>
-          <Add />
+        <Button
+          variant="contained"
+          color="primary"
+          size="large"
+          startIcon={<Add />}
```
Comment (aninternetof) [inline src/components/ProgramList.js:198]: There is special support for leading icons.

File: src/index.js [hunk 2]
```diff
@@ -85,14 +86,16 @@ render(
...
         <CookiesProvider>
-          <Switch>
-            <Route path="/accounts" component={Accounts} />
-            <ProtectedRoute exact path="/" component={ProgramList} />
-            <ProtectedRoute exact path="/programs" component={ProgramList} />
-            <ProtectedRoute exact path="/mission-control" component={MissionControl} />
-            <ProtectedRoute exact path="/user/settings" component={UserSetting} />
-            <Route component={NotFound} />
-          </Switch>
+          <ThemeProvider theme={responsiveFontSizes(theme)}>
```
Comment (aninternetof) [inline src/index.js:89]: `responsiveFontSizes` will auto adjust our relative font sizes based on screen width. They can also be set manually for different screen widths, so we'll have to see how we like the defaults.

image

File: src/theme.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+export default createMuiTheme({
+  palette: {
+    primary: {
+      main: '#114BFD', /* blue */
+    },
+    secondary: {
+      main: '#FF0459', /* magenta */
+    },
+    error: {
+      main: '#FF0000', /* TODO: some kind of red??? */
```
Comment (aninternetof) [inline src/theme.js:12]: Need to get an error color from UX.

File: src/theme.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    warning: {
+      main: '#FEAD11', /* yellow */
+    },
+    info: {
+      main: '#00FDA3', /* light green */
+    },
+    success: {
+      main: '#00FF00', /* TODO: some kind of green??? */
```
Comment (aninternetof) [inline src/theme.js:21]: Need to get a success color from UX.

File: src/theme.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    contrastThreshold: 3,
+    tonalOffset: 0.2,
+  },
+  typography: {
+    fontFamily: [
+      'Lato', 'sans-serif',
+    ].join(','),
+    button: {
+      fontWeight: 'bold',
+      textTransform: 'none',
```
Comment (aninternetof) [inline src/theme.js:32]: Material Design calls for all-caps buttons, but let's undo that.

Comment (aninternetof) [reply src/theme.js:32]: The bad hover-colors on our buttons is from some Semantic-UI CSS. That should do away when we completely remove Semantic.

File: src/index.js [hunk 6]
```diff
@@ -12,11 +13,11 @@ import { createLogger } from 'redux-logger';
 import promise from 'redux-promise-middleware';
 import createDebounce from 'redux-debounced';
 import * as Sentry from '@sentry/browser';
+import theme from '@/theme';
```
Comment (cabarnes) [inline src/index.js:16]: I'm thinking we might have more themes in the future. What do you think about putting this in a `themes` directory with this named `default` in `default.js`?

Comment (aninternetof) [reply src/index.js:16]: Great idea.

Comment (cabarnes) [reply src/index.js:16]: Maybe just `light`?

File: src/index.js [hunk 7]
```diff
@@ -12,11 +13,11 @@ import { createLogger } from 'redux-logger';
 import promise from 'redux-promise-middleware';
 import createDebounce from 'redux-debounced';
 import * as Sentry from '@sentry/browser';
+import rovercodeDefaultLight from '@/themes/rovercodeDefaultLight';
```
Comment (cabarnes) [inline src/index.js:16]: I think you can put an `index.js` in `themes/` exporting each of the themes so you can do `import { rovercodeDefaultLight } from '@/themes;` and we can import multiple in the future in one import

Comment (aninternetof) [reply src/index.js:16]: Hm, that's giving me: `"export 'default' (imported as 'light') was not found in '@/themes'`

Comment (aninternetof) [reply src/index.js:16]: Using `export const light = createMuiTheme({` in src/themes/index.js and `import light from '@/themes`

Comment (cabarnes) [reply src/index.js:16]: I think in `index.js` it would be something like this:
```
import light from light;
export default {
  light,
}
```

Comment (aninternetof) [reply src/index.js:16]: ah, thanks. Still coming up to speed on js modules.

Comment (aninternetof) [reply src/index.js]: Landed on this strategy: https://alligator.io/react/index-js-public-interfaces/

**Pull request merged**